src/test/java/tests/SearchJira.java

```diff
@@ -108,6 +114,58 @@ public void epmtyResultsIssue() {
         searchPage.iconEpmtyResults();
     }
 
+    @Test
+    public void testSaveFilter(){
+        dashboardPage.clickIssueButton();
+        dashboardPage.clickSearchOfIssues();
+        searchPage.clickfindFiltersButton();
+        manageFiltersPages.clickMyButton();
+        manageFiltersPages.deleteFilterIfExist("1 testSaveFilter");
+        dashboardPage.clickIssueButton();
+        dashboardPage.clickSearchOfIssues();
+        searchPage.clickSearchProjectButton();
+        searchPage.selectProjectQAAUTO6("QAAUTO-6");
```

Comment:
Why do you use an argument "QAAUTO-6" if your  method's name is "selectProjectQAAUTO6"?

Reply:
I have already rewritten this method and added searchPage.selectProject. This is typo. So i will change.